src/main/java/org/jabref/gui/EntryType.fxml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.*?>
+<?import java.lang.*?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+
+<DialogPane xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1" fx:controller="org.jabref.gui.EntryTypeView">
+   <content>
+       
+      <BorderPane>             
+         <bottom>
+            <Button fx:id="cancelButton" text="Cancel" BorderPane.alignment="CENTER" />
+         </bottom>
+         <center>
+            <GridPane alignment="CENTER" BorderPane.alignment="CENTER">
+               <children>
+                  <Label text="ID type" />
```

Comment:
Every text that is localized needs to be prepanded by `%`, e.g. here `text="%ID type"`